.github/CODEOWNERS

```
@@ -47,6 +47,7 @@ airflow-core/src/airflow/ui/public/i18n/locales/nl/ @BasPH # + @DjVinnii
 airflow-core/src/airflow/ui/public/i18n/locales/pl/ @potiuk @mobuchowski # + @kacpermuda
 airflow-core/src/airflow/ui/public/i18n/locales/zh-TW/ @Lee-W @jason810496 # + @RoyLee1224 @guan404ming
 airflow-core/src/airflow/ui/public/i18n/locales/fr/ @pierrejeambrun @vincbeck
+airflow-core/src/airflow/ui/src/i18n/locales/tr/ @bugraoz93
```

Still wrong folder!
```suggestion
airflow-core/src/airflow/ui/public/i18n/locales/tr/ @bugraoz93
```

---

Should @hasancatalgol also be added as "Engaged Translator"?
```suggestion
airflow-core/src/airflow/ui/public/i18n/locales/tr/ @bugraoz93 # +@hasancatalgol
```

---

@bugraoz93 @jscheffl yes please, planning to support in the long-term